### PR TITLE
corrected incorrect links

### LINKED ISSUE #1357: multiple dead links in admission_check.md
**What happened**:
multiple invalid links are present 
**What you expected to happen**:
All links should work
**How to reproduce it (as minimally and precisely as possible)**:
click on link
**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
- Kueue version (use `git describe --tags --dirty --always`):
- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

### PR DESCRIPTION
**Pull request by kundan2707**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:
Corrected multiple incorrect link
#### Which issue(s) this PR fixes:

Fixes #1357 

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Review by tenzen-y — COMMENTED** [review]
Thanks for your contribution!

File: site/content/en/docs/concepts/admission_check.md [hunk 1]
```diff
@@ -86,4 +86,4 @@ The logic for how an AdmissionCheck changes states is not part of Kueue.
 
 ## What's next?
 
-- Read the [API reference](/docs/reference/kueue.v1beta1/#kueue-x-k8s-io-v1beta1-AdmissionCheck) for `AdmissionCheck`
+- Read the [API reference](/docs/reference/kueue.v1beta1.md#kueue-x-k8s-io-v1beta1-AdmissionCheck) for `AdmissionCheck`
```
[the remark this answers is not in the document]
Comment (tenzen-y) [reply site/content/en/docs/concepts/admission_check.md:89]: The current link should work fine.

[the remark this answers is not in the document]
Comment (kundan2707) [reply site/content/en/docs/concepts/admission_check.md:89]: I checked current one was not working and giving file not found page

File: site/content/en/docs/concepts/admission_check.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ description: >
 
 AdmissionChecks are a mechanism that allows Kueue to consider additional criteria before admitting a Workload.
 When a ClusterQueue has AdmissionChecks configured, each of the checks has to provide a
-positive signal to the Workload before it can be [Admitted](docs/concepts/#admission).
+positive signal to the Workload before it can be admitted.
```
[the remark this answers is not in the document]
Comment (kundan2707) [reply site/content/en/docs/concepts/admission_check.md:12]: @tenzen-y  I will make this change

File: site/content/en/docs/concepts/admission_check.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (tenzen-y) [reply site/content/en/docs/concepts/admission_check.md:89]: Really? Doesn't this page work fine?

https://kueue.sigs.k8s.io/docs/concepts/admission_check/

[the remark this answers is not in the document]
Comment (kundan2707) [reply site/content/en/docs/concepts/admission_check.md:89]: @tenzen-y sorry its working find i will revert changes

**Pull request merged**